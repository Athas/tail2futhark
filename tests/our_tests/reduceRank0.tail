
let v0:[int]1 = [1,2,3,4,5,6] in
let v1:[int]0 = reduce{[int],[0]}(addi,0,v0) in
v1
