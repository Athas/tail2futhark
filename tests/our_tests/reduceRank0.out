fun real main() =
  let t_v0 = [1,2,3,4,5,6] in
  let t_v1 = reduce(op +,0,t_v0) in
  toReal(t_v1)